### PR TITLE
Management

### PR DESCRIPTION
**Pull request by MistzSoftware**
This pull request includes significant updates to the `README.md` and `SECURITY.md` files, enhancing the project's documentation and updating security support information.

Documentation updates in `README.md`:

* Added new sections for Gameplay Overview, Features, Project Objectives, Art and Design, Community and Feedback, Roadmap, and Future Plans to provide comprehensive information about the game and its development.
* Improved text formatting for better readability, including bolding key terms and adding emphasis to the game's title and character names.
* Updated version information to reflect the latest and current versions of the project.

Security updates in `SECURITY.md`:

* Updated the supported versions table to indicate that versions 0.1.0 and 0.2.0 are now supported.

### CODE REVIEW

**Review by Fus1onAxel — APPROVED** [review]
approved

File: README.md [hunk 1]
```diff
@@ -28,21 +28,64 @@ Read the list of [contributors](https://github.com/algosup/2024-2025-project-2-s
 
 ## 📚 What is this project ?
 
-**Forsaken Planet** is a serious game developed in Godot 4.3, focused on the theme of climate change. In the game, players will explore various environments on a mysterious planet, solving puzzles and uncovering clues about the disappearance of an ancient civilization that once thrived there.
+***Forsaken Planet*** is a serious game developed in Godot 4.3, focused on the theme of climate change. In the game, players will explore various environments on a mysterious planet, solving puzzles and uncovering clues about the disappearance of an ancient civilization that once thrived there.
```
Comment (Furimizu) [inline README.md:31]: maybe highlight godot

File: README.md [hunk 2]
```diff
@@ -28,21 +28,64 @@ Read the list of [contributors](https://github.com/algosup/2024-2025-project-2-s
...
 
-As Quireo, an explorer from the planet Eradon, your mission is to uncover the fate of the ancient civilization and gather crucial information to prevent similar catastrophic events from occurring on your home planet. Along the way, you will be assisted by your colleague and friend, Thothan a brilliant scientist whose knowledge and insight will be vital in your quest for answers.
+As **Quireo**, an explorer from the planet Eradon, your mission is to uncover the fate of the ancient civilization and gather crucial information to prevent similar catastrophic events from occurring on your home planet. Along the way, you will be assisted by your colleague and friend, **Thothan**, a brilliant scientist whose knowledge and insight will be vital in your quest for answers.
```
Comment (Furimizu) [inline README.md:33]: maybe highlight Eradon

File: README.md [hunk 3]
```diff
@@ -28,21 +28,64 @@ Read the list of [contributors](https://github.com/algosup/2024-2025-project-2-s
...
+
+## 🌟 Features
+
+* **Immersive Environments**: Lush forests, barren wastelands, and technologically advanced ruins brought to life with Godot's 4.3 rendering capabilities.
+* **Dynamic Interaction**: Real-time dialogue and task coordination with Thothan to solve problems.
+* **Educational Elements**: Subtle learning moments designed to challenge perceptions about climate change and foster critical thinking.
+
+## 📈 Project Objectives
+
+We actively want to develop a game that blends **entertainment with impactful ecological education**, encourage players to reflect on climate change and consider actionable solutions as well as deliver an engaging narrative that **resonates with both casual and avid gamers**.
```
Comment (Furimizu) [inline README.md:56]: maybe "think" instead of "reflect"

**Review by Furimizu — APPROVED** [review]

**Pull request merged**